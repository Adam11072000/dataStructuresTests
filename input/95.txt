init 33
deleteImage 31
resetLabelScore 101 27 231
getHighestScoredLabel 245 -4
mergeSuperPixels 231 -8 28
setLabelScore 116 29 -2 204
addImage 191
resetLabelScore 191 7 0
mergeSuperPixels 191 3 0
addImage -5
addImage 0
quit