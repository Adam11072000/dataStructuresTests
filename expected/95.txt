init done.
deleteImage: FAILURE
resetLabelScore: FAILURE
getHighestScoredLabel: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
addImage: INVALID_INPUT
addImage: INVALID_INPUT
quit done.
